app/src/main/assets/pages/Insuree.js

```diff
@@ -54,9 +54,9 @@ $(document).ready(function () {
             $('#imgInsuree').attr('src', 'file://' + ImagePath);
         } else {
             $('#imgInsuree').attr('src', '');
-            $("#hfNewPhotoPath").val("");
         }
 
+        $("#hfNewPhotoPath").val("");
```

Comment:
why setting it twice to ''' ; see line 12

Reply:
This assignment is done during insuree number field's change event. The other one (line 12) is a part of page initialization.